test/test_advisory.py

```diff
@@ -158,3 +158,26 @@ def test_reporter_cant_edit_advisory(db, client):
     assert Forbidden.code == resp.status_code
     assert_advisory_data(DEFAULT_ADVISORY_ID, workaround='yay', impact='meh')
     assert 1 == advisory_count()
+
+
+def test_advisory_get_impact_from_text(db, client):
+    impact = advisory_get_impact_from_text(DEFAULT_ADVISORY_CONTENT)
+    assert 'Robots will take over' in impact
+    assert 'Impact' not in impact
+    assert 'References' not in impact
+
+    assert advisory_get_impact_from_text('test') == None
+
+
+def test_advisory_get_workaround_from_text(db, client):
+    content='\nWorkaround\n==========\n\nUpdate your machine\n\nDescription\n'
+    impact = advisory_get_workaround_from_text(DEFAULT_ADVISORY_CONTENT)
+    assert 'Update your machine' in impact
+    assert 'Workaround' not in impact
+    assert 'Description' not in impact
+
+    assert advisory_get_workaround_from_text('test') == None
+
+    content='\nWorkaround\n==========\n\nNone.\n\nDescription\n'
+    impact = advisory_get_workaround_from_text(content)
+    assert advisory_get_workaround_from_text('test') == None
```

Comment:
what are those lines doing? content and impact is not used anymore and the assert is identical to the previous one

Reply:
Hrrmpfft indeed, that looks wrong. But the two lines should test the case where there is no workaround:
```
    if 'None.' == workaround: 
        return None 
```